include/irr/asset/IAsset.h

```diff
@@ -128,8 +170,14 @@ class SAssetBundle
         assert(allSameTypeAndNotNull());
     }
 
+	//! Returns an Asset type associated with the Asset
```

Comment:
Bundle can only hold an array of assets that are the same type

Reply:
status?

Reply:
Solved